WTWSMS/common/minor_titles/00_minor_titles.txt

```diff
@@ -770,11 +762,13 @@ title_caesar = {
 		OR = {
 			has_landed_title = e_byzantium
 			has_landed_title = e_roman_empire
+			has_landed_title = e_wre
 		}
 		OR = {
 			religion_group = christian
 			religion_group = jewish_group
 			religion_group = pagan_group
+			religion = bosporan_faith
```

Comment:
@loup99 

Bosporans should give the title because Jews and Pagans should give the title.